lib/rules/v-on-handler-style.js

```diff
@@ -300,6 +305,36 @@ module.exports = {
         return false
       }
 
+      // disable the auto-fixed when the node does't have params
+      if (
+        !hasComment /* The statement contains comment and cannot be fixed. */ &&
+        idCallExpr /* The statement is not a simple identifier call and cannot be fixed. */ &&
+        idCallExpr.arguments.length === 0
+      ) {
+        const paramCount = methodParamCountMap.get(idCallExpr.callee.name)
```

Comment:
I think this rule also has the number of arguments defined in the method. If we check that, I think some code can be safely fixed automatically.
If it's unsafe or we can't identify the number of arguments for the method, I think it's good to make it a suggestion.